src/chm/documents/customactions/wixnetfxextension.html.md

```diff
@@ -7,122 +7,6 @@ after: using_standard_customactions
 
 The [WixNetfxExtension](~/xsd/netfx/index.html) includes a set of custom actions to compile native images using Ngen.exe. For an example, see [How To: NGen managed assemblies during installation](~/howtos/files_and_registry/ngen_managed_assemblies.html).
 
-## PackageGroups
```

Comment:
I think I should have linked to the how-to from here.  It doesn't make sense to me for bundle information like this to be under the `customactions` folder, I don't think that's discoverable either.  Maybe create a NetFxExtension page under the `bundles` folder for this information?